src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -2,20 +2,26 @@
 
 import com.revrobotics.CANSparkLowLevel.MotorType;
 import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.constants.RobotConstants.IntakeConstants;
 
 public class Intake extends SubsystemBase {
   private final CANSparkMax m_rollerMotor; // Intake roller motor
+  private final DigitalInput m_linebreak;
 
   // Constructs intake and initializes motor
   public Intake() {
     m_rollerMotor = new CANSparkMax(IntakeConstants.kMotorID, MotorType.kBrushless);
+    m_linebreak = new DigitalInput(IntakeConstants.kLineBreakSensor);
```

Comment:
Shouldn't this control the state of the command?

Reply:
current implementation only verifies the line break sensor reading changes

Reply:
In the future, we'll put in on shuffleboard

Reply:
Where is the logic that verifies the reading changes? Also putting it on shuffleboard doesn't really address the question. I thought the point of the sensor was to make it so that the intake would stop once a note has been indexed -- is that still the plan? If so, this logic should be updated to reflect that. If not, or if it's coming in the future, let's tag it with a TODO so we don't lose track of it.